x-pack/plugins/apm/server/routes/metrics/by_agent/serverless/compute_usage.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import {
+  FAAS_BILLED_DURATION,
+  FAAS_COLDSTART_DURATION,
+  METRIC_SYSTEM_TOTAL_MEMORY,
+} from '../../../../../common/elasticsearch_fieldnames';
+import { Setup } from '../../../../lib/helpers/setup_request';
+import { fetchAndTransformMetrics } from '../../fetch_and_transform_metrics';
+import { ChartBase } from '../../types';
+
+const chartBase: ChartBase = {
+  title: i18n.translate('xpack.apm.agentMetrics.serverless.computeUsage', {
+    defaultMessage: 'Compute usage',
+  }),
+  key: 'cold_start',
+  type: 'linemark',
+  yUnit: 'number',
+  series: {
+    computeUsage: {
+      title: i18n.translate('xpack.apm.agentMetrics.serverless.computeUsage', {
+        defaultMessage: 'Compute usage',
+      }),
+    },
+  },
+};
+
+const computeUsageScript = {
+  lang: 'painless',
+  source: `
+    if(doc.containsKey('${METRIC_SYSTEM_TOTAL_MEMORY}') && doc.containsKey('${FAAS_BILLED_DURATION}')){
+      double faasBilledDurationValue =  doc['${FAAS_BILLED_DURATION}'].value;
+      double totalMemoryValue = doc['${METRIC_SYSTEM_TOTAL_MEMORY}'].value;
+      return totalMemoryValue * faasBilledDurationValue
```

Comment:
I think we need to adapt the unit here (or at other locations), since the computeUsage needs to be reported in `GB-seconds`. 
The unit of `totalMemoryValue` is `Bytes` and the unit of `faasBilledDurationValue` is `milliseconds`.

So we need to divide this value here by `1024*1024*1024*1000`